lib/redux/middleware/local_storage_middleware.dart

```diff
@@ -19,9 +19,11 @@ class LocalStorageMiddleware extends MiddlewareClass<AppState> {
       final packageInfo = await PackageInfo.fromPlatform();
       final version = 'Version ${packageInfo.version} (${packageInfo.buildNumber})';
       store.dispatch(RehydrateSuccessAction(authToken, version));
+      store.dispatch(FetchAuthenticatedUserAction());
```

Comment:
Do we really need to refetch the user every time the app starts up?

- User email might change, but not id?
- Doesn't the Sentry SDK persist the user info?